packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -126,6 +127,13 @@ function validateClassInstance(inst, methodName) {
  * @lends ReactTestUtils
  */
 function renderIntoDocument(element) {
+  if (disableDOMTestUtils) {
+    throw new Error(
+      '`renderIntoDocument` was removed from `react-dom/test-utils`. ' +
+        'See https://react.dev/warnings/react-test-renderer for more info.',
```

Comment:
TODO: Write docs

Reply:
Should we generalize this warning page to something like `/warnings/testing` and include sections for each separate warning between ReactTestUtils and react-test-renderer?

Reply:
I'd probably have a page for test-renderer and test-utils

Reply:
Docs: https://github.com/reactjs/react.dev/pull/6716